linkup/src/lib.rs

```diff
@@ -64,7 +65,7 @@ pub fn get_additional_headers(
     }
 
     let tracestate = headers.get("tracestate");
-    let linkup_session = format!("linkup-session={}", session_name);
+    let linkup_session = format!("linkup-session={}", session_name,);
```

Comment:
This is me not pretending to know what it would mean to add a `,` here.
Is it intentional?

Reply:
Quite right, I'll fix it next time!